### DISCUSSION TITLE
nominatim replication command need  manualy import ?

### DISCUSSION THREAD

**Discussion by MALKARAJ** [mediagis/nominatim-docker#372] [Q&A]
Hi Team,
Thanks for doing the great work keep it up,

i have an another doubt ,
By updating the data using nominatim replication service , whether indexing will done automatically or we need to initialize the indexing manually after every new update

**Comment by mtmail** [comment]
The replication will do the indexing automatically, no need to do anything extra. By default it uses only CPU thread and thus runs in the background without interfearing with user's running API queries.
